src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -154,10 +194,10 @@ public static String getListingComments(SscsCaseData caseData) {
         }
         if (nonNull(otherParties) && !otherParties.isEmpty()) {
             listingComments.addAll(otherParties.stream()
-                                       .map(CcdValue::getValue)
-                                       .filter(o -> isNotBlank(o.getHearingOptions().getOther()))
-                                       .map(o -> getComment(o, o.getHearingOptions().getOther()))
-                                       .collect(Collectors.toList()));
+                    .map(CcdValue::getValue)
+                    .filter(o -> isNotBlank(o.getHearingOptions().getOther()))
+                    .map(o -> getComment(o, o.getHearingOptions().getOther()))
+                    .collect(Collectors.toList()));
```

Comment:
This can be improved to something like
```suggestion
                    .map(CcdValue::getValue)
                    .map(OtherParty::getHearingOptions)
                    .map(HearingOptions::getOther)
                    .filter(StringUtils::isNotBlank)
                    .map(this::getComment)
                    .collect(Collectors.toList()));
```

Reply:
Reverted this change as it wasn't working